gstruct/elements.go

```diff
@@ -13,10 +13,14 @@ import (
 
 //MatchAllElements succeeds if every element of a slice matches the element matcher it maps to
 //through the id function, and every element matcher is matched.
-//  Expect([]string{"a", "b"}).To(MatchAllElements(idFn, matchers.Elements{
-//      "a": BeEqual("a"),
-//      "b": BeEqual("b"),
-//  })
+//    idFn := func(element interface{}) string {
```

Comment:
I took some liberties here. Let me know if I should dial this back and simplify, but I thought a copy/paste-able example was helpful

Reply:
LGTM